### PR TITLE
Shift preferences and Offer shifts documentation

### PR DESCRIPTION
**Pull request by akumar39**
Only shift preferences completed.
Offer shifts WIP

### CODE REVIEW

File: api-reference/beta/api/offershiftrequest-approve.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+author: "akumar39"
+ms.prod: "microsoft-teams"
+doc_type: "apiPageType"
+---
+
+# offerShiftRequest: approve
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+PROVIDE DESCRIPTION HERE
```
Comment (Lauragra) [inline api-reference/beta/api/offershiftrequest-approve.md]: Add the missing description here and in the YAML header.

File: api-reference/beta/api/offershiftrequest-approve.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+## Permissions
+
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
+
+| Permission type                        | Permissions (from least to most privileged) |
+|:---------------------------------------|:--------------------------------------------|
+| Delegated (work or school account)     | Group.ReadWrite.All |
+| Delegated (personal Microsoft account) | Not supported. |
+| Application                            | Schedule.ReadWrite.All (currently in Private Preview) |
```
Comment (Lauragra) [inline api-reference/beta/api/offershiftrequest-approve.md]: Please document the private preview permissions the same way that we did in the other method topics. For an example, see: https://docs.microsoft.com/en-us/graph/api/team-put-schedule?view=graph-rest-beta&tabs=http#permissions .

File: api-reference/beta/api/offershiftrequest-decline.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+## Permissions
+
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
+
+| Permission type                        | Permissions (from least to most privileged) |
+|:---------------------------------------|:--------------------------------------------|
+| Delegated (work or school account)     | Group.ReadWrite.All |
+| Delegated (personal Microsoft account) | Not supported. |
+| Application                            | Schedule.ReadWrite.All (currently in Private Preview) |
```
Comment (Lauragra) [inline api-reference/beta/api/offershiftrequest-decline.md]: Global: Please make the same update for all permissions.

File: api-reference/beta/api/offershiftrequest-decline.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+author: "akumar39"
+ms.prod: "microsoft-teams"
+doc_type: "apiPageType"
+---
+
+# offerShiftRequest: decline
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+PROVIDE DESCRIPTION HERE
```
Comment (Lauragra) [inline api-reference/beta/api/offershiftrequest-decline.md]: Add the description.

File: api-reference/beta/resources/changetrackedentity.md [hunk 5]
```diff
@@ -1,43 +1,68 @@
...
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
+Represents an entity to track changes made to any supported [schedule](schedule.md) and associated resource.
+
+## Methods
+
+| Method       | Return Type | Description |
+|:-------------|:------------|:------------|
+| [Get](../api/changetrackedentity-get.md) | [changeTrackedEntity](changetrackedentity.md) | Read the properties and relationships of a **changeTrackedEntity** object. |
```
Comment (Lauragra) [inline api-reference/beta/resources/changetrackedentity.md]: We don't have topics for these operations - pls. add.

File: api-reference/beta/resources/offershiftrequest.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+Represents type of shift request to offer a shift to another user in the team.
+
+## Methods
+
+| Method       | Return Type | Description |
+|:-------------|:------------|:------------|
+| [Create](../api/offershiftrequest-post.md) | [offerShiftRequest](offershiftrequest.md) | Create an instance of an offerShiftRequest object. |
+| [Get](../api/offershiftrequest-get.md) | [offerShiftRequest](offershiftrequest.md) | Read properties and relationships of offerShiftRequest object. |
+| [List](../api/offershiftrequest-list.md) | Collection of [offerShiftRequest](offershiftrequest.md) | Read properties and relationships of all offerShiftRequest objects in a team. |
+| [Update](../api/offershiftrequest-update.md) | [offerShiftRequest](offershiftrequest.md) | Update offerShiftRequest object. |
```
Comment (Lauragra) [inline api-reference/beta/resources/offershiftrequest.md]: We're missing method topics for update and delete; pls. add.

File: api-reference/beta/api/swapshiftschangerequest-list.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
+---
+title: "List swapShiftsChangeRequest"
+description: "Retrieve the properties and relationships of swapShiftsChangeRequest object in the team."
+localization_priority: Normal
+author: "akumar39"
+ms.prod: "microsoft-teams"
+doc_type: "apiPageType"
+---
+
+# Get swapShiftsChangeRequest
```
Comment (Lauragra) [inline api-reference/beta/api/swapshiftschangerequest-list.md]: Please verify whether this is a list or a get operation.

File: api-reference/beta/api/shiftpreferences-put.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+            },
+            "timeZone": "Pacific Standard Time",
+            "timeSlots": null
+        }
+    ]
+}
+```
+
+#### Response
+
```
Comment (millicentachieng) [inline api-reference/beta/api/shiftpreferences-put.md]: @Lauragra, @akumar39 The response needs to be annotated
```
<!-- {
  "blockType": "response", 
  "@odata.type": "microsoft.graph.None"
} -->
```

**Pull request merged**